js/src/modules/_MobileMenu.es6.js

```diff
@@ -23,7 +24,7 @@ class _MobileMenu {
     menuButtonClass = 'mobile-menu-button--menu', // Class name for generated menu button
     closeButtonClass = 'mobile-menu-button--close', // Class name for generated close button
     arrowButtonClass = 'mobile-menu__subnav-arrow', // Class name for the subnav toggle
-    mobileMenuBreakpoint = '(max-width: 699px)', // Breakpoint to switch between mobile + original menu
+    mobileMenuBreakpoint = `(max-width: ${BREAKPOINTS['mobile-menu']})`, // Breakpoint to switch between mobile + original menu
```

Comment:
1) No, they should not. That's a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).
2) I used max-width deliberately here because the way the mobile menu works in Gesso, it's an addition to the default functionality. The mobile menu is created via JS and feels to me like a form of progressive enhancement. In particular, https://github.com/forumone/gesso/blob/8.x-3.x/js/src/modules/_MobileMenu.es6.js#L204 makes more sense to me using max-width, rather than writing a script to implement that functionality and then to undo it on larger screens. I agree that defaulting to min-width unless there's a reason to do otherwise is the right rule of thumb, but since the intention behind that, as I understand it, is to avoid serving unnecessary CSS to smaller screens with limited bandwidth, I will deviate from that in places where min-width results in more code all around.